src/builds/travis.rs

```diff
@@ -0,0 +1,97 @@
+use std::io::Read;
+
+use chrono::{DateTime, UTC};
+use diesel;
+use diesel::prelude::*;
+use hyper::Client;
+use hyper::header::UserAgent;
+use hyper::net::HttpsConnector;
+use hyper_native_tls::NativeTlsClient;
+use serde::de::DeserializeOwned;
+use serde_json;
+
+use DB_POOL;
+use domain::builds::Build;
+use error::DashResult;
+
+header! { (Accept, "Accept") => [String] }
+
+const ROOT_URL: &'static str = "https://api.travis-ci.org";
+const UA: &'static str = "rusty-dash/0.0.0";
```

Comment:
...So I don't actually know what Travis looks for in a UA (I chose it kind of arbitrarily). All it says in the docs are:

> Always set the User-Agent header. This header is not required right now, but will be in the near future. Assuming your client is called “My Client”, and it’s current version is 1.0.0, a good value would be MyClient/1.0.0. For our command line client running on OS X 10.9 on Ruby 2.1.1, it might look like this: Travis/1.6.8 (Mac OS X 10.9.2 like Darwin; Ruby 2.1.1; RubyGems 2.0.14) Faraday/0.8.9 Typhoeus/0.6.7.